dev_guide/deployments.adoc

```diff
@@ -191,6 +191,53 @@ To see what the rollback would look like without performing the rollback:
 $ oc rollback <deployment_config> --dry-run
 ----
 
+[[executing-commands-inside-a-container-deployments]]
+== Executing Commands Inside a Container
```

Comment:
I suppose the discrepency between the anchor id and the title is due to the topic title being appended, to avoid potential conflict when other sections w/ identical title are created.  But, i wonder, is that possibility likely?